docker/webui/run_openqa.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# prepare environment
+rm -f /run/dbus/pid
+/usr/share/openqa/script/upgradedb --user geekotest --upgrade_database
+
+# run services
+dbus-daemon --system --fork
+su geekotest -c '/usr/share/openqa/script/openqa-scheduler &'
+su geekotest -c '/usr/share/openqa/script/openqa-websockets &'
+su geekotest -c '/usr/share/openqa/script/openqa gru -m production run &'
+rcapache2 start && su geekotest -c '/usr/share/openqa/script/openqa prefork -m production --proxy'
```

Comment:
the tools will have to be kept around as long as we maintain sysv compatibility. You are right, start_daemon doesn't fork by itself, you'd still have to use '&' to use shell job control. To avoid that, startproc could be used.
An alternative to the sysv tools would be chroot I guess. It can also switch uid.
Can't the docker image just run systemd inside?